modules/games/definitions/outwit/instructions.ts

```diff
@@ -0,0 +1,24 @@
+import { OutwitDefinition } from "./_types";
+
+const outwitInstructions: OutwitDefinition["instructions"] = {
+  startTurn: { line: ["Select a piece to move"] },
```

Comment:
Strings matching the name of a unit group will be replaced with an icon for those units, which is kind of sweet!

```suggestion
  startTurn: {
    line: [
      "Select a"
      "soldiers",
      "or",
      "kings",
      "to move"
    ]
  },
```

Writing a unit group like `"kings"` here is a shorthand for `{ unittype: ["kings", ["player"]] }`, where the second argument is the colour the unit should be shown in. `["player"]` is the current player, which is what will be used when we just write `"kings"`.

Reply:
Ahh neat